app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupMakingFragment.java

```diff
@@ -113,6 +114,7 @@ public void onNameListsImported(Map<String, Integer> nameAmounts, String listNam
     @OnClick(R.id.make_groups)
     void makeGroups() {
         if (listInfo.getNumNames() == 0) {
+        UIUtils.showLongToast(R.string.group_no_names_error_message,getContext());
```

Comment:
nit: Spacing is a bit off.
```suggestion
            UIUtils.showLongToast(R.string.group_no_names_error_message,getContext());
```